makefile

```diff
@@ -240,7 +241,9 @@ install_DIRS += $(_DESTDIR)$(sysconfdir)/cron.daily
 #
 default_INSTCMD = cp
 $(_DESTDIR)$(sysconfdir)/cron.daily_TARGETS        = pkg/cron.daily:serviced
+$(_DESTDIR)$(sysconfdir)/cron.hourly_TARGETS       = pkg/cron.hourly:serviced
 $(_DESTDIR)$(prefix)/etc_TARGETS                   = pkg/serviced.logrotate:logrotate.conf
+$(_DESTDIR)$(prefix)/etc_TARGETS                  += pkg/logrotate-docker.conf:logrotate-docker.conf
```

Comment:
It could go there, then you would have one more thing to depend on.  By adding our own scripts to cron.daily and cron.hourly, we only have to ensure that crond is running in the docs.